libsplinter/src/node_id/mod.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
All of the copy rights should be updated to `// Copyright 2018-2021 Cargill Incorporated`

Reply:
Should it be updated project wide? I fixed the ones in here